src/SFA.DAS.EmployerIncentives.Commands.Types/ApprenticeshipIncentive/CreateCommand.cs

```diff
@@ -0,0 +1,37 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Abstractions.Logging;
+using SFA.DAS.EmployerIncentives.Domain.Accounts;
+using SFA.DAS.EmployerIncentives.Infrastructure.DistributedLock;
+using System;
+
+namespace SFA.DAS.EmployerIncentives.Commands.Types.ApprenticeshipIncentive
+{
+    public class CreateCommand : ICommand, ILockIdentifier, ILogWriter
```

Comment:
Create what?

Reply:
It's in the ApprenticeshipIncentive namespace so create an ApprenticeshipIncentive.  Does it need to be more wordy?

Reply:
I think so as it isn't immediately obvious what it does.

Reply:
Done